### PR TITLE
Add tests.

### PR DESCRIPTION
**Pull request by mzhang28**
#34

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        conn.execute(DropConstraint(fkc))
+
+    for table in tbs:
+        conn.execute(DropTable(table))
+
+    trans.commit()
+
+
+@pytest.fixture(scope="session")
+def app(request):
```
Comment (chaosagent) [inline conftest.py:9]: Config not set to testing here; csrf protection won't be disabled

Comment (mzhang28) [reply conftest.py:9]: Will I need to load CalendarOptions too? I thought I wouldn't because it was already done in cal.py

Comment (chaosagent) [reply conftest.py:9]: Yeah.
app.config.from_object(CalendarConfig(testing=true)) or whatever it was

File: conftest.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+@pytest.fixture(scope="session")
+def client(app):
+    return app.test_client()
+
+
+@pytest.fixture(scope="class")
+def db(request, app):
+    cal_db.reflect()
+    drop_everything(cal_db)
```
Comment (chaosagent) [inline conftest.py]: is drop_everything actually necessary?
reflect and drop_all works pretty much every time for me

Comment (mzhang28) [reply conftest.py]: idk i just copied it over from the other thing cuz it didn't work there
want me to remove it and see how it goes?

**Comment by chaosagent** [comment]
Looks good to me

**Pull request merged**